ide-support/revsblibrary.livecodescript

```diff
@@ -2875,6 +2875,15 @@ private command revSmartChecking pStackFilesList, @xSettings
    unlock messages
 end revSmartChecking
 
+private function utilityScriptIsLocked pStack
+   try
+      get the script of stack pStack
+      return false
+   catch tError
+      return true
+   end try
+end utilityScriptIsLocked
+
```

Comment:
For other private utilities in this file we use the convention of beginning the handler name with double underscore, so I suggest just calling this `__ScriptIsLocked`.
I would also put the `return false` after the try I think